packagers/pnpm-workspaces-default-output/test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+FILE1=sub-project-1/node_modules/.prisma/client/index.js
```

Comment:
Why only check for one of them here?

Reply:
Actually, thanks for pointing this out because it would fail. It should be
```
FILE1=node_modules/.prisma/client/index.js
```
And check that it *does not* exist in the sub-folders. This is something I tried to change before and it caused regressions. Therefore I added a test for it. When no specific location is specified, the client is generated near `@prisma/client`, in this case in the hoisted `node_modules`.